test/WebJobs.Script.Tests.Integration/WebJobs.Script.Tests.Integration.csproj

```diff
@@ -61,6 +61,13 @@
     <ProjectReference Include="..\..\src\WebJobs.Script.WebHost\WebJobs.Script.WebHost.csproj" />
     <ProjectReference Include="..\..\src\WebJobs.Script.Grpc\WebJobs.Script.Grpc.csproj" />
     <ProjectReference Include="..\TestFunctions\TestFunctions.csproj" Private="true" />
+    <ProjectReference Include="..\CSharpPrecompiledTestProjects\AssemblyLoadContextRace\AssemblyLoadContextRace.csproj" ReferenceOutputAssembly="false" />
```

Comment:
You can use a glob if you want:

``` xml
    <ProjectReference Include="..\CSharpPrecompiledTestProjects\**\*.csproj" ReferenceOutputAssembly="false" />
```

You also will need to add to the solution, otherwise VS build breaks (VS tries to be all fancy with its build, I don't like it 🤣 )

I like to use dotnet cli for that;
From the repo root:

``` pwsh
Get-ChildItem .\test\CSharpPrecompiledTestProjects\ -Recurse -Filter *.csproj | % { dotnet sln WebJobs.Script.sln add $_.FullName }
```

Reply:
Thank you for this, will update